phpunit.xml.dist

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-    bootstrap="vendor/autoload.php"
-    colors="true"
-    columns="max"
-    verbose="true"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+        bootstrap="vendor/autoload.php"
+        colors="true"
+        columns="max"
+        verbose="true"
 >
-    <testsuites>
-        <testsuite name="Faker Test Suite">
-            <directory>./test/Faker/</directory>
-        </testsuite>
-    </testsuites>
+  <testsuites>
+    <testsuite name="Faker Test Suite">
+      <directory>./test/Faker/</directory>
+    </testsuite>
+  </testsuites>
```

Comment:
All of the changes in this file are unrelated, can we revert them, please?

Reply:
reverted